iati/core/resources/test_data/202/valid_startswith.xml

```diff
@@ -1,6 +1,21 @@
 <?xml version="1.0"?>
-
+<!--all elements and attributes contain the specified prefixing string-->
 <root_element>
-  <reporting-org ref="text"></reporting-org><!-- iati-identifier starts with expected prefix -->
-  <iati-identifier>text-1234</iati-identifier>
+  <condition></condition><!--condition case-->
+  <element1>prefix1234</element1><!--single path-->
+  <element2>prefix5678</element2><!--multiple paths-->
+  <element3>prefixpuppies</element3>
+  <element4>prefixkittens</element4><!--duplicate paths-->
+  <element5 attribute="prefixbunnies"></element5><!--single path-->
+  <element6 attribute="prefixapples"></element6><!--multiple paths-->
+  <element7 attribute="prefixfrogs"></element7>
+  <element8 attribute="prefixflowers"></element8><!--duplicate paths-->
+  <nest>
+    <element9>prefixtoads</element9><!--nested path-->
+  </nest>
+  <nest>
+    <element10 attribute="prefixbeardeddragonsarecool"></element10><!--nested path-->
+  </nest>
+  <element11>prefix0123</element11><!--duplicate element-->
+  <element11>prefix4567</element11>
```

Comment:
Suggest having one that is only `prefix` to ensure it's not checking for `prefix[\w\d]+`

Reply:
done